postgres/DockerfileFullExt

```diff
@@ -0,0 +1,25 @@
+FROM mdillon/postgis:11
```

Comment:
ensure using `Dockerfile` as the name and move it to proper sub folder

Reply:
I think that's the best I can do for restructuring; see my new commits
and for the full extension tag, I need to reuse some files from other tags